arch/cpu/nrf/os/dbg-arch.c

```diff
@@ -48,31 +48,41 @@
 #include "contiki.h"
 
 #include "uarte-arch.h"
+#include "usb.h"
 /*---------------------------------------------------------------------------*/
 unsigned int
 dbg_send_bytes(const unsigned char *s, unsigned int len)
 {
+#if PLATFORM_DBG_CONF_USB
+  usb_write((uint8_t *)s, len);
```

Comment:
I suggest handling the flush separately from the write here as well, similar to:
https://github.com/contiki-ng/contiki-ng/blob/629b374eafb850caba7dc6d0330caa336b2ab037/arch/cpu/nrf52840/dbg.c#L83

Reply:
Thank you. Done.